Flake8 complaining after release

Flake8 had a release on 3/29/25 and emits the error:
```
./src/aspire/nufft/__init__.py:118:5: F824 `global backends` is unused: name is never assigned in scope
check: exit 1 (1.44 seconds) /Users/carmichael/Work/ASPIRE-python.coord_source> flake8 . pid=245
  check: FAIL code 1 (1.48=setup[0.04]+cmd[1.44] seconds)
  evaluation failed :( (1.54 seconds)
```

I'll take a look and create a patch.

--- a/src/aspire/nufft/__init__.py
+++ b/src/aspire/nufft/__init__.py
@@ -115,8 +115,6 @@ def all_backends():
 
     :return: A list of strings representing available NFFT backends
     """
-    global backends
-
     if backends is None:
         check_backends(raise_errors=False)
     return [k for k, v in backends.items() if v is not None]

